app/views/admin/reports/_status.html.haml

```diff
@@ -18,7 +18,7 @@
       - if status.application
         = status.application.name
         ·
-      = link_to ActivityPub::TagManager.instance.url_for(status), class: 'detailed-status__datetime', target: stream_link_target, rel: 'noopener noreferrer' do
+      = link_to admin_account_status_path(status.account.id, status), class: 'detailed-status__datetime' do
```

Comment:
I believe this change is consistent with our other changes of "no unsafe links for moderators" (see the https://github.com/mastodon/mastodon/pull/27139 for background).